packages/synapse-react-client/src/synapse-client/SynapseClient.ts

```diff
@@ -2053,18 +2053,16 @@ export const getFileResult = (
 }
 
 /**
- * Add a file to the user's download list.
+ * Add a batch of files to the user's download list.
  * Uses http://rest-docs.synapse.org/rest/POST/download/list/add.html
- * @param fileEntityId
- * @param versionNumber
+ * @param batchToAdd
  */
-export const addFileToDownloadListV2 = (
-  fileEntityId: string,
-  versionNumber?: number,
+export const addFileBatchToDownloadListV2 = (
```

Comment:
Added new service to request a batch of files. `addFileToDownloadListV2` was refactored to call this with its single file batch.